run.sh

```diff
@@ -72,6 +72,12 @@ case "$deployment_action" in
         build_images
         deploy_osh
         ;;
+    "setup_airship")
```

Comment:
Great to add this, can this be in a different commit though?

Reply:
> Great to add this, can this be in a different commit though?

To test setup_airship, I have to update the inventory hosts file and add groups that airship requires, so the two changes kind of go hand in hand. I could add this in a separate commit but wont be able to test it.

Reply:
Went ahead and created separate PR to add 'setup_airship' as per your suggestion. This PR is again getting too long and might be good to just restrict it to inventory changes.

The other PR to add setup_airship is https://github.com/SUSE-Cloud/socok8s/pull/165 I will rebase this once that PR is merged.

Reply:
Done. #165 was merged. Rebased this PR.